tmt/utils.py

```diff
@@ -532,11 +539,29 @@ def workdir(self):
             create_directory(self._workdir, 'workdir', quiet=True)
         return self._workdir
 
+    @staticmethod
```

Comment:
IMO not necessary - cli/plan merging might happen and tmt should take care of it.